components/library/libwebm/libwebm.p5m

```diff
@@ -0,0 +1,36 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file usr/bin/$(MACH64)/sample path=usr/demo/libwebm/$(MACH64)/sample
+file path=usr/bin/$(MACH64)/sample_muxer
+file path=usr/bin/$(MACH64)/vttdemux
+file path=usr/bin/$(MACH64)/webm2pes
+
+file usr/bin/sample path=usr/demo/libwebm/sample
+file path=usr/bin/sample_muxer
+file path=usr/bin/vttdemux
+file path=usr/bin/webm2pes
+
+file path=usr/lib/$(MACH64)/libwebm.so
+file path=usr/lib/libwebm.so
```

Comment:
If we want just to have tools, I'd rather compiled library as static one and avoided delivering its files. Can you recall the software which required external libwebm?